atlasdb-dbkvs/src/main/java/com/palantir/atlasdb/keyvalue/dbkvs/impl/postgres/PostgresDdlTable.java

```diff
@@ -128,7 +135,7 @@ public void truncate() {
     public void checkDatabaseVersion() {
         AgnosticResultSet result = conns.get().selectResultSetUnregisteredQuery("SHOW server_version");
         String version = result.get(0).getString("server_version");
-        PostgresVersionCheck.checkDatabaseVersion(version, log);
```

Comment:
Why not modify the method to take a `SafeLogger` instead? If you're worried about public API, you can always keep the old version of the method